src/main/java/kira/storage/SaveLoad.java

```diff
@@ -0,0 +1,105 @@
+package kira.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+import kira.exception.KiraException;
+import kira.task.Deadline;
+import kira.task.Event;
+import kira.task.Task;
+import kira.task.TaskType;
+import kira.task.ToDo;
+
+/**
+ * SaveLoad handles all the saving of tasklist and loading of tasklist into the bot.
+ */
+public class SaveLoad {
+
+    /**
+     * Saves current tasks list to a CSV file.
+     *
+     * @throws KiraException File-Output-Issue
+     */
+    public static void save(List<Task> taskList, String pathname) throws KiraException {
+        try {
+            File saveFile = new File(pathname);
+            FileWriter fileWriter = new FileWriter(saveFile);
+            for (Task t : taskList) {
+                fileWriter.write(t.saveFormat());
+                fileWriter.write("\n");
+            }
+            fileWriter.flush();
+            fileWriter.close();
+        } catch (IOException e) {
+            throw new KiraException("Error encountered when saving file :C");
+        }
+    }
+
+    /**
+     * Loads and parses the csv save file into a list of tasks.
+     *
+     * @param pathname path where the csv file is stored
+     * @return a list of tasks
+     * @throws KiraException File-Not-Found
+     */
+    public static List<Task> load(String pathname) throws KiraException {
+        List<Task> taskList = new ArrayList<>();
```

Comment:
```suggestion
        List<Task> taskLists = new ArrayList<>();
```
As this is a list which represents a collection of objects, plural form should be used. I noticed the same instances in several other places too.